ng2-components/ng2-alfresco-documentlist/src/components/document-list.component.html

```diff
@@ -30,11 +30,10 @@
         </no-content-template>
     </div>
 </alfresco-datatable>
-<alfresco-pagination *ngIf="enablePagination"
+<alfresco-pagination *ngIf="enablePagination && !isEmpty()"
```

Comment:
Can you create a single method that does this check? 
Something like isEnabledPagination()